index.html

```diff
@@ -36,5 +36,104 @@ <h1>I'm Norman<br> Glad to see you!</h1>
             </ul>
         </div>
     </section>
+    <section class="works-section">
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work1.png" alt="work 1">
+            </div>
+            <div class="work-details">
+                <h1>Tonic</h1>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <p class="project-btn"><a href="">See Project</a></p>
+            </div>
+        </div>
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work2.png" alt="work 2">
+            </div>
+            <div class="work-details">
+                <h1>Multi-Post<br> Stories</h1>
+                <p class="work-subtext">CANOPY<span class="dot"></span><span class="description">Back End Dev</span><span class="dot"></span><span class="description">2015</span> </p>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <p class="project-btn"><a href="">See Project</a></p>
+            </div>
+        </div>        
+        <div class="work-card">
+            <div class="work-image">
+                <img src="images/work3.png" alt="work 3">
+            </div>
+            <div class="work-details">
+                <h1>Tonic</h1>

```

Comment:
- [ ] I suggest removing the `<h1>` tag from here and using another header tag `hx` (eg `<h2>`,  `<h3>`, ...) instead. to Follow best practices for HTML. **Only Use H1 Elements Once Per Page for optimal SEO**. for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/).

> Note: you already used `<h1>` here ``` <h1>I'm Norman<br> Glad to see you!</h1>```, that's great because this is the main title 💯

- [ ] Note: Please don't forget to do this on all project titles 👌